recipe/meta.yaml

```diff
@@ -75,12 +70,12 @@ outputs:
         - test -f ${PREFIX}/lib/liblz4.a      # [unix]
         - if not exist %LIBRARY_LIB%\\liblz4_static.lib exit 1  # [win]
 
-
 about:
   home: https://lz4.github.io/lz4/
   license: BSD-2-Clause
   license_family: BSD
   license_file: lib/LICENSE
+  license_family: BSD
```

Comment:
`license_family` is duplication of https://github.com/AnacondaRecipes/lz4-c-feedstock/pull/3/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR76